_data/sidebars/user_docs.yml

```diff
@@ -23,14 +23,19 @@ entries:
       url: /quickstart
       output: web, pdf
 
-    - title: Build an Image
+    - title: Build a Container
       url: /docs-build
       output: web, pdf
 
     - title: Container Recipes
       url: /docs-recipes
       output: web, pdf
 
+
+    - title: Singularity Flow
+      url: /docs-flow
+      output: web, pdf
+
     - title: Singularity Flow
```

Comment:
Why is this here twice?

Reply:
Can we remove one?

Reply:
Oh crap.  That was probably an error when I did the merge.  I'll look again.